<launch>

  <arg name="filename" />

  <rosparam param="teach_motions/filename" subst_value="True">
    $(arg filename)
  </rosparam>

  <rosparam command="load" file="$(find teach_motions)/cfg/$(arg filename).yaml" />

  <node name="compliant_replay_server" pkg="teach_motions" type="compliant_replay_server" output="screen" />

</launch>
